ext/pdo_mysql/pdo_mysql.stub.php

```diff
@@ -33,7 +33,7 @@ class Mysql extends \PDO
     /** @cvalue PDO_MYSQL_ATTR_COMPRESS */
     public const int ATTR_COMPRESS = UNKNOWN;
 
-    /** @cvalue PDO_MYSQL_ATTR_DIRECT_QUERY */
+    /** @cvalue PDO_ATTR_EMULATE_PREPARES */
     public const int ATTR_DIRECT_QUERY = UNKNOWN;
 
     /** @cvalue PDO_MYSQL_ATTR_FOUND_ROWS */
```

Comment:
Maybe we should remove this from 8.4 as it is a new class and barely out? @php/release-managers-84 what do you think?

Reply:
PHP 8.4 is out already. Maybe it would be better to drop in 8.5?

Reply:
Yes, but I would consider this a "fix" to remove this useless constant, and considering it would be early in the release of it I'm not sure that it is that problematic, we fixed various named parameters up until PHP 8.0.3 to make them consistent.